ajax/libs/sticky-js/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "sticky-js",
+  "filename": "dist/sticky.min.js",
+  "description": "Sticky-js is a library for sticky elements written in vanilla javascript. With this library you can easily set sticky elements on your website. It's also responsive.",
+  "license": "MIT",
+  "author": "rgalus <biuro@rafalgalus.pl>",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/rgalus/sticky-js.git"
+  },
+  "keywords": [
+    "sticky",
+    "position",
+    "scroll",
+    "javascript"
+  ],
+  "autoupdate": {
```

Comment:
Let's use npm auto-update instead. There are more versions on npm compared to git repo.
It would be like
```javascript
  "npmName": "sticky-js",
  "npmFileMap": [
    {
      "basePath": "dist",
      "files": [
        "**/!(*.gz)"
      ]
    }
  ]
```
Use the pattern `**/!(*.gz)` can prevent `sticky.min.js.gz`, which we don't need, from being fetched.
For more details about auto-update, you can see [autoupdate.md](https://github.com/cdnjs/cdnjs/blob/master/documents/autoupdate.md)